config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
   devise_for :users
   root 'texts#index'
+  resources :texts
```

Comment:
ルーティングは必要なものに制限するようにしていきましょう。

```suggestion
  resources :texts, only: [:index, :show]
```